veangle-guides/locking-angle.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Guide to lock ANGLE and get veANGLE
+---
+
+# Locking ANGLE into veANGLE
+
+ANGLE holders will be able to lock their tokens between 1 week and 4 years to obtain veANGLE, granting them voting power in the new governance system, boost on rewards for some staking contracts, and a share of the interests generated by the protocol.
+
+Before locking ANGLE, make sure you read all the details about veANGLE and locking [here](../governance/veANGLE/README.md).
+
+## Guide
+
+ANGLE tokens can be locked at [dao.angle.money](https://dao.angle.money).
+
+Before locking, remember that only one lock per address can be created. Once an address initiates a lock, it can either increase the lock expiration, or add tokens to the lock.
+
+### Creating the lock
+
+To create a lock, an expiration date and a quantity of tokens need to be specified. Then, two transactions are needed, first to approve the contract and then to create the lock with the specified amount of tokens.
+
+![Locking veANGLE](../.gitbook/assets/locking-veangle.png)
+
+#### Locking ANGLE before staking
+
+One particularity of the gauge system is that the staking contracts can’t fetch the veANGLE balance of stakers in realtime. Therefore, this information needs to be updated once in a while by the staking contracts. This happens at checkpoints, when users stake, unstake, or claim rewards from the gauge.
+
+In practice, this means that it’s better to lock ANGLE into veANGLE **before staking** in order for the boost to be applied directly. If not, clamining a tiny amount of rewards **after** staking will be needed to apply the boost.
+
+To compute the balance of veANGLE needed to get the max boost on rewards depending on the quantity of tokens staked, there is a calculator at [dao.angle.money](https://dao.angle.money).
+
+### Increasing the lock or adding tokens
+
+Once the lock has been created, to extend the lock expiration or add tokens, just click on one of the two buttons and choose a new expiration date (up to 4 years from now) or a quantity of tokens to add.
+
+![Increasing or adding tokens to lock](../.gitbook/assets/increase-add-lock.png)
+
+### Claiming interests
+
+Every week, a share of the interest generated by the protocol is redistributed with veANGLE holders.
```

Comment:
We should say here that interests are given under the form of sanUSDC_EUR at the moment